flink-end-to-end-tests/test-scripts/test_streaming_file_sink.sh

```diff
@@ -44,14 +41,16 @@ if [ "${OUT_TYPE}" == "local" ]; then
 elif [ "${OUT_TYPE}" == "s3" ]; then
   echo "Use s3 output"
   JOB_OUTPUT_PATH=${S3_OUTPUT_PATH}
+  set_config_key "state.checkpoints.dir" "s3://$IT_CASE_S3_BUCKET/$OUT-chk"
+  mkdir -p "$OUTPUT_PATH-chk"
```

Comment:
there's no reason why `common_s3` (or the minio variant for that matter) should control where stuff is written to on disk

Reply:
I don't quite get it. This is the actual test, where we also want to test if checkpoints on s3 work with plugins (spoiler alert: they don't on the current master).

Reply:
I think I mixed something up, please ignore